cmd/cmd.go

```diff
@@ -59,6 +60,24 @@ func GetMeshConfig(kube kubernetes.Interface, namespace, name string) (*proxycon
 		return nil, multierror.Prefix(err, "failed to convert to proto.")
 	}
 
+	if mesh.MixerAddress != "" && mesh.StatsdUdpAddress == "" {
+		colon := strings.Index(mesh.MixerAddress, ":")
+		mixerAddr := mesh.MixerAddress[:colon]
+		if err := model.ValidateIPv4Address(mixerAddr); err == nil {
+			mesh.StatsdUdpAddress = mesh.MixerAddress
+		} else {
+			if mixerSvc, err := kube.CoreV1().Services(namespace).Get(mixerAddr, v1.GetOptions{}); err == nil {
```

Comment:
cmd/ should not be depending on k8s. I'd prefer we did not automatically set values in the config since it obscures problems and makes debugging difficult.